.github/workflows/terraform-checks.yaml

```diff
@@ -16,7 +16,7 @@ on:
         description: The channel that the charm is deployed from. Defaults to `latest/stable`.
```

Comment:
please change the description to match :)
```suggestion
        description: The channel that the charm is deployed from. Defaults to `latest/edge`.
```

Reply:
Updated